### ISSUE TITLE
Support OpenEXR 3.0

### ISSUE THREAD

**Issue by 64** [cessen/openexr-rs#48]
AFAICT OpenEXR 3 no longer includes Ilm, so the build script fails.

**Comment by cessen** [comment]
Yes, this is definitely something we want to support.  Since it's a major release with breaking changes, however, it's not entirely clear to me if it's feasible to support within the same crate.  I'll investigate this when I get the chance.
